src/shared/components/lazyMobXTable/LazyMobXTable.spec.tsx

```diff
@@ -398,6 +399,25 @@ describe('LazyMobXTable', () => {
         clock.uninstall();
     });
 
+    describe('downloadData', () => {
+        it.only('only displays columns with defined download functions', () => {
```

Comment:
Oop, you left an `only`
```suggestion
        it('only displays columns with defined download functions', () => {
```